names/rom_wasserman.txt

```diff
@@ -0,0 +1,4 @@
+Rom Wasserman
+rom.png
```

Comment:
Did you test this? I don't think this will work, because your image is named `rom.png.PNG` and not `rom.png`.

![image](https://user-images.githubusercontent.com/26610904/115120467-aef4ba80-9f7b-11eb-8035-457884c897f9.png)

Please let me know if you have tested this and confirm that it works -- I believe you will have to update the file name here though to match.